dlc/README.md

```diff
@@ -0,0 +1 @@
+#TODO
```

Comment:
The other modules don't really have meaningful READMEs, FYI. Probably makes more sense to move this to the website docs, adding a section on DLCs